Rubberduck.Core/UI/IOpenFileDialog.cs

```diff
@@ -108,7 +108,20 @@ public bool DereferenceLinks
 
         public void Dispose()
         {
+            Dispose(true);
```

Comment:
Was confused to see an implementation of `Dispose` in a file named `IOpenFileDialog` - turns out that the file has both `IOpenFileDialog` and `OpenFileDialog`. If we're going to keep both interface and implementation in one file, I think it's more logical to call it after the class, not the interface.

Reply:
Agreed, wasn't even looking at the file names when I was burning through here.  Might as well rename these since I'm touching them.